src/components/atoms/ToggleRP/ToggleRP.tsx

```diff
@@ -45,7 +46,14 @@ export class ToggleRP extends React.Component<ToggleProps, ToggleState> {
       <label className={classNames('ToggleRP', this.props.className)}>
         <input
           id={'test-id-input'}
-          onClick={this.props.disabled ? noop : callAll(this.props.onClick, this.toggle)}
+          onClick={
+            this.props.disabled
```

Comment:
I had to change this lines because in development mode does not work. Sincerely I don't know why. The state never update and throw this error.

```
Warning: Can't call setState on a component that is not yet mounted. This is a no-op, but it might indicate a bug in your application. Instead, assign to `this.state` directly or define a `state = {};` class property with the desired state in the ToggleRP component.
```

I researched this issue, but I didn't find any reason yet.